QuantLib/ql/termstructures/volatility/optionlet/optionletstripper.hpp

```diff
@@ -40,6 +40,7 @@ namespace QuantLib {
       public:
         //! \name StrippedOptionletBase interface
         //@{
+        enum Model{Normal, ShiftedLognormal};
```

Comment:
I can make the change with regards to the "case naming". But just to be sure we have no misunderstanding ...
**Normal Case**
![equation](http://latex.codecogs.com/gif.latex?dF%3D%5Csigma%20dW)
**Shifted LogNormal Case**
![equation](http://latex.codecogs.com/gif.latex?d%28F%20+%20a%29%3DdF%3D%28F+a%29%5Csigma%20dW)

where *a* is the shift (i.e. displacement within the constructor). So there is no shift in the normal case whereas the shifted LogNormal case contains the usual LogNormal case with *a=0* as a special case. This has been the reason for my naming convention.